spot-ml/ml_ops.sh

```diff
@@ -104,5 +104,6 @@ time spark-submit --class "org.apache.spot.SuspiciousConnects" \
   --threshold ${TOL} \
   --maxresults ${MAXRESULTS} \
   --ldamaxiterations 20 \
+  --scalingOption ${SCALING_OPTION} \
```

Comment:
"scaling option" is not the correct way to describe what is going... we are not doing any "scaling" which would be mapping the values to some fixed interval (eg. by multiplying by some scaling factor)

The range of values is [0,1] whether we use float or doubles. What varies is the precision, how many low-order bits we can get 

"scoring precision" or "model precision" just "precision" would be more appropriate